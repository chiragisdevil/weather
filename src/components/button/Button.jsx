// Displays a styled button. Takes children to support different text

import React from "react";
import "./button.scss";

function Button({children}){
    return(
        <button className= "custom-button">{children}</button>
    )
}

export default Button;