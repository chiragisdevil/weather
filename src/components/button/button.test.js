// import { EnzymeAdapter } from "enzyme";
import {shallow} from "enzyme";
import Button from "./Button";

it("checks if button component matches the snapshot", () => {
    expect(shallow(<Button> Start Again </Button>)).toMatchSnapshot();
})
