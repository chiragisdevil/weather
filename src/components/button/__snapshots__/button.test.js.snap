// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if button component matches the snapshot 1`] = `ShallowWrapper {}`;
